theia-java-docker/next.package.json

```diff
@@ -15,20 +15,89 @@
         "@theia/file-search": "next",
         "@theia/getting-started": "next",
         "@theia/git": "next",
-        "@theia/java": "next",
         "@theia/json": "next",
```

Comment:
I think maybe we're keeping this one because the json built-in is not working yet?

Reply:
Correct, the `vscode-builtin-json-language-features` is not yet working so I could not add it at this time (https://github.com/eclipse-theia/theia/issues/6623).

`vscode-builtin-json` is included to add JSON syntax highlighting and snippet support since `@theia/textmate` grammars is removed.